modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -47,6 +47,14 @@
                 expect(response.location).to include(expected)
               end
             end
+
+            it 'calls shared validation' do
+              mock_ccg(scopes) do |auth_header|
+                expect_any_instance_of(ClaimsApi::V2::DisabilityCompensationValidation)
+                  .to receive(:validate_form_526_submission_values!)
+                post submit_path, params: data, headers: auth_header
+              end
+            end
```

Comment:
since we have tests to catch the validation logic in the async flow, we had discussed leaving just the assurance that synchronous uses the same validation. that way they stay "in line" with the requests coming in and we can have confidence that our updates won't break a validation error. 

One caveat is this is for the "validate" ticket so assuming `submit_path` needs changed....